slam_toolbox/src/slam_toolbox_common.cpp

```diff
@@ -586,6 +586,11 @@ bool SlamToolbox::serializePoseGraphCallback(
   slam_toolbox_msgs::SerializePoseGraph::Response &resp)
 /*****************************************************************************/
 {
+  if (!smapper_->getMapper()->IsLoopClosed())
+  {
+    return false;
```

Comment:
Add a log here for “hey, couldnt serialize”

Reply:
Ok, I am thinking to add this:
`ROS_WARN("Could not serialize the map because no loop closure has been detected yet!");`

Reply:
That works